src/modules/person/domain/username-generator.service.ts

```diff
@@ -71,9 +71,11 @@ export class UsernameGeneratorService {
             return calculatedUsername;
         }
         let counter: number = 1;
+        /* eslint-disable no-await-in-loop */
```

Comment:
The solution that will be in the Email-Generator can be used here.

Reply:
as discussed opened in ticket SPSH-1081